packages/extension-chrome/src/contentScript/content.ts

```diff
@@ -1,5 +1,5 @@
 import { isJSONRPCRequest, isJSONRPCResponse, JSONRPCClient } from 'json-rpc-2.0';
-import * as bridgeMessenger from 'webext-bridge';
+import { onMessage, sendMessage } from 'webext-bridge/content-script';
```

Comment:
```suggestion
import * as bridgeMessenger from 'webext-bridge/content-script';
```

there are two messengers in this file, and to avoid confusion, it is best to distinguish between them with different names

Reply:
updated